aioshelly/block_device.py

```diff
@@ -407,6 +407,10 @@ async def set_state(self, **kwargs: Any) -> dict[str, Any]:
             "get", f"{self.type}/{self.channel}", kwargs
         )
 
+    async def set_state_full_path(self, path: str, **kwargs: Any) -> dict[str, Any]:
```

Comment:
This is now not really a function that sets a block state but a function that calls a generic REST command for the device.

I had the same comment from @balloob about RPC device that the name should match what it does.

In my opinion we should consider:
1. Name it something more related to what it does: `http_get_request` or `call_http_get`
2. Since it is not tied to a block now (unlike `set_state` which sets state within the path of a block) we should probably move it to the device class (same where `http_request` is located)

Reply:
Actually I don't completely agree as this is just a different way to set the block.